set.mm

```diff
@@ -182295,17 +182295,16 @@ infinite descent (here implemented by strong induction).  This is
        Metamath 100 proof #1.  (Contributed by NM, 20-Aug-2001.)  (Revised by
        Mario Carneiro, 12-Sep-2015.) $)
```

Comment:
```suggestion
       Metamath 100 proof #1.  (Contributed by NM, 20-Aug-2001.)  (Proof
       shortened by Jannik Vierling, 4-Jan-2022.) $)
```

Reply:
Usually, we keep the "Revised" tag, don't we? Therefore, we should have:

... (Contributed by NM, 20-Aug-2001.)  (Revised by Mario Carneiro, 12-Sep-2015.) (Proof shortened by Jannik Vierling, 4-Jan-2022.) $)

Reply:
I think we only keep 2 credits, the first one and the latest change. I'm okay with doing it either way.

Reply:
I merged without realizing this suggestion was not added :(  I or @jvierling can add it if you're ok with that.  We'll also add your name to the "contributor list" at the top of the file, so that the credit can also be `(Proof shortened by JV, 4-Jan-2022.)`.

By the way, @digama0 : is there an easy tool to add one contributor to the three-column list (every line changes to keep alphabetical order...)?  If not, maybe we can make it single column ? (This time, it's ok since "V" is near the end of the list.)

Reply:
> I think we only keep 2 credits, the first one and the latest change. I'm okay with doing it either way.

In my opinion, we should keep one tag of each kind (Contributed, Revised, Proof shortened). If a theorem is revised, the "Proof shortened" tag can be removed, since the proof would be changed because of the revision anyway.

Reply:
@benjub You mean the contributor list with the initials? Pretty sure that was maintained manually. @avekens Let's do that.

Reply:
Since this PR is already merged, I'll open a new pull request with the required changes.

Reply:
As for the attribution tags: I would vote for keeping, obviously, the earlierst (which is the "contributed by") and also the latest. For the other ones, it should be on a case-by-case analysis, depending on the importance of the shortening/revision (especially when the latter is documented); if there were two important successive revisions, both should be kept.